### PR TITLE
ports system/schema to tycho

### PR DESCRIPTION
**Pull request by ymo-sci**
Bildschirmfoto 2021-11-11 um 15 38 05
Bildschirmfoto 2021-11-11 um 15 37 58
Bildschirmfoto 2021-11-11 um 15 37 38

### CODE REVIEW

File: src/main/resources/default/templates/biz/model/schema.html.pasta [hunk 1]
```diff
@@ -1,52 +1,63 @@
...
+            </div>
         </div>
     </div>
-
     <script type="text/javascript">
 
         function executeChange() {
-            var $row = $(this).parent().parent();
-            $row.addClass("info");
+            let $row = $(this).parent().parent();
```
Comment (sabieber) [inline src/main/resources/default/templates/biz/model/schema.html.pasta:43]: Don't want we to switch to vanilla JS where possible in tycho?

File: src/main/resources/default/assets/styles/biz-misc.scss [hunk 2]
```diff
@@ -14,3 +14,8 @@
   max-height: 400px;
   overflow-y: auto;
 }
+
```
Comment (andyHa) [inline src/main/resources/default/assets/styles/biz-misc.scss:17]: nope, we use declarative styles / classes now...

Comment (andyHa) [reply src/main/resources/default/assets/styles/biz-misc.scss:17]: also, these seem to be default values(?)

File: src/main/resources/default/templates/biz/model/schema.html.pasta [hunk 3]
```diff
@@ -1,52 +1,63 @@
...
-                    </th>
-                </tr>
-                </thead>
-                <tbody id="schemaList">
-                </tbody>
-            </table>
+        <div class="col">
+            <div class="card">
+                <div class="card-body">
+                    <table id="schemaTable" class="table table-sm table-striped">
```
Comment (andyHa) [inline src/main/resources/default/templates/biz/model/schema.html.pasta:22]: in Tycho we commonly only use class="table"

File: src/main/resources/default/templates/biz/model/schema.html.pasta [hunk 4]
```diff
@@ -56,45 +67,52 @@
...
-                                row.addClass("warning");
-                                row.append($('<td class="align-right"><a href="#" class="link link-danger">@i18n("DatabaseController.execute")</a></td>'));
+                                row.children(":first").addClass("border-sirius-yellow");
+                                row.append($('<td class="align-right"><a href="#" class="btn btn-outline-danger">@i18n("DatabaseController.execute")</a></td>'));
                             }
                             $('a', row).click(executeChange);
                             row.appendTo($('#schemaList'));
                         }
+                    } else {
+                        $('#schemaList').html('<tr><td colspan="3" class="align-center">'
```
Comment (andyHa) [inline src/main/resources/default/templates/biz/model/schema.html.pasta:113]: I'd kille the whole table and replicate the look&feel of t:emptyCheck here....

**Review by andyHa — CHANGES_REQUESTED** [review]

**Review by sabieber — APPROVED** [review]

**Pull request merged**